Mathlib/GroupTheory/Perm/DomMulAct.lean

```diff
@@ -106,6 +107,7 @@ theorem stabilizer_card [DecidableEq α] [DecidableEq ι] [Fintype ι] :
   · rfl
 
 /-- The cardinality of the set of permutations preserving a function -/
+@[nolint finiteFintype]
```

Comment:
What is the reason that this nolint is necessary? The `Fintype` instance is used directly in the `∏` notation.

Reply:
Good question, I'm not sure. @urkud Do you know?

Reply:
You don't need `[Fintype α]`, only `[Finite α]` here. In fact, I'm not sure if you need `[Finite α]` at all: if `α` is infinite, then one of the fibers is infinite, and RHS = 0. OTOH, LHS is infinite or empty, so you don't need `[Finite α]`.

Reply:
@urkud Do you have a moment to try to make this change? I won't, until next week.

Reply:
Fixed.

Reply:
Thanks!

Reply:
Ah, I thought it was complaining about the iota fintype instance so it looked like a bug. I didn't realize there was a `[Fintype α]` instance too. Thanks for the clarification (and for improving the lemma!)